feat: youtube embedd component

vinckr opened the pull request:
## Related issue

<!--
Please link the GitHub issue this pull request resolves in the format of `#1234`. If you discussed this change
with a maintainer, please mention her/him using the `@` syntax (e.g. `@aeneasr`).

If this change neither resolves an existing issue nor has sign-off from one of the maintainers, there is a
chance substantial changes will be requested or that the changes will be rejected.

You can discuss changes with maintainers either in the Github Discusssions in this repository or
join the [ORY Chat](https://www.ory.sh/chat).
-->

## Proposed changes

<!--
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
-->

## Checklist

<!--
Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
-->

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

## Further comments

<!--
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution
you did and what alternatives you considered, etc...
-->

vinckr commented:
@jfcurran is going to add the Grid

aeneasr reviewed (changes requested):
I assume this isn't the final version? Because, I mean...

<img width="1420" alt="Bildschirmfoto 2021-06-18 um 15 55 18" src="https://user-images.githubusercontent.com/3372410/122571991-9bce9b80-d04d-11eb-942c-8e3767dd7c07.png">

vinckr commented:
Too big? 

I thought about making it pop out from a CTA `Why Hydra` or similar, but for the release really big so its not to be missed.

aeneasr commented:
Yes, it is way too big, does not fit the flow of the website, and is lacking context. The video is great but I should at least know what I’m watching before I click it - so for example „Use Case Sainsbury‘s“ on the left and vid on the right or something.

vinckr commented:
Still fiddling a bit with the text, but how does it look?

aeneasr commented:
Better! I would probably put it below the mail list banner and above the easy integration. The layouts should then be switched so right left / left right / right left for a nice visual flow

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
       />
+
+<FeatureImage
+        overline={'A proven solution'}
```
```suggestion
        overline={'Proven Scalability'}
```

aeneasr replied:
Devs don't really like the word "solution". I think we have proven scalability because Sainsbury's has a lot of users. Therefore, that's a claim we can make here! For solution, it's not clear what it solves. The reader has to make the connection between million of users, the video.

Also, from an SEO perspective, "proven solution" says nothing. However, if I search for "scalable oauth2 server" then this might help!

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
+        overline={'A proven solution'}
+        title={<>Millions of users? No problem</>}
+        description={
+          <>
+            If you need a reliable open source solution Hydra is the right tool for the job.
+            <br></br>Watch the video to learn how Hydra powers Sainsbury's My ID, handling hundreds of users per second with ease.
+          </>
```
Please no `<br>` tags, keep it all in one line and let the layout engine decide how to break the text.

```suggestion
            If you need a reliable open source solution Hydra is the right tool for the job. Watch the video to learn how Hydra powers Sainsbury's My ID, handling hundreds of users per second with ease.
          </>
```

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
+
+<FeatureImage
+        overline={'A proven solution'}
+        title={<>Millions of users? No problem</>}
```
```suggestion
        title={<>Scale to Millions</>}
```

aeneasr replied:
We don't really use punctuation elsewhere :)

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
+        title={<>Millions of users? No problem</>}
+        description={
+          <>
+            If you need a reliable open source solution Hydra is the right tool for the job.
+            <br></br>Watch the video to learn how Hydra powers Sainsbury's My ID, handling hundreds of users per second with ease.
```
Try to avoid "If you need" and make things imperative and positive.

> Sainsbury's ... with millions of customers, serving hundreds of users per second. Replicate their success now: **link**

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
+        buttons={
+          <>
+            <Button to={'/docs/hydra/'} style={'link'} iconRight={ArrowRight}>
+              Check out the docs
```
```suggestion
              Get started
```

aeneasr replied:
I think you might want to get started once you saw the video!

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
+        }
+        buttons={
+          <>
+            <Button to={'/docs/hydra/'} style={'link'} iconRight={ArrowRight}>
```
link to get started

aeneasr reviewed (changes requested):
meant to request changes

vinckr commented:
thanks for the quick feedback :)
adressed your requested changes and also removed the `/` from Ory / Products, as its the only place where its written like that. 
Also added contraste, an adopter of hydra & oathkeeper

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,16 +75,42 @@ const IndexPage = () => {
+        title={<>Scale to millions</>}
+        description={
+          <>
+            Learn how Ory Hydra powers Sainsbury's My ID, with millions of
+            customers, serving hundreds of users per second. Replicate their
+            success now.
```
```suggestion
            Ory Hydra powers Sainsbury's My ID with millions of
            customers and serving hundreds of users per second.
```

aeneasr reviewed (commented):
Making it a bit leaner :)

aeneasr commented on src/pages/hydra.tsx:
```diff
@@ -74,16 +75,41 @@ const IndexPage = () => {
+        description={
+          <>
+            Ory Hydra powers Sainsbury's My ID with millions of
+            customers and serving hundreds of users per second.
```
```suggestion
            customers and serving hundreds of users per second.
            Watch the video to learn more!
```
